documentation/sphinx/source/backups.rst

```diff
@@ -102,7 +102,7 @@ If <secret> is not specified, it will be looked up in :ref:`blob credential sour
 
 An example blob store Backup URL would be ``blobstore://myKey:mySecret@something.domain.com:80/dec_1_2017_0400?bucket=backups``.
 
-Blob store Backup URLs can have optional parameters at the end which set various limits or options used when communicating with the store.  All values must be positive decimal integers unless otherwise specified.  The speed related default values are not very restrictive.  The most likely parameter a user would want to change is ``max_send_bytes_per_second`` (or ``sbps`` for short) which determines the upload speed to the blob service.
+Blob store Backup URLs can have optional parameters at the end which set various limits or options used when communicating with the store.  All values must be positive decimal integers unless otherwise specified.  The speed related default values are not very restrictive. A parameter is applied individually on each `backup_agent`, meaning that a global restriction should be calculated based on the number of agent running. The most likely parameter a user would want to change is ``max_send_bytes_per_second`` (or ``sbps`` for short) which determines the upload speed to the blob service. 
```

Comment:
Looks like build failed for the error
```
[844/1834] cd /codebuild/output/src3814492293/src/github.com/apple/foundationdb/build_output/documentation && /usr/local/bin/cmake -E make_directory /codebuild/output/src3814492293/src/github.com/apple/foundationdb/build_output/documentation/html && /opt/rh/rh-python38/root/usr/local/bin/sphinx-build -W -b html -d doctree -c /codebuild/output/src3814492293/src/github.com/apple/foundationdb/documentation/sphinx /codebuild/output/src3814492293/src/github.com/apple/foundationdb/documentation/sphinx/source /codebuild/output/src3814492293/src/github.com/apple/foundationdb/build_output/documentation/html -Dversion=7.4.0 -Drelease=OFF && /usr/local/bin/cmake -E touch /codebuild/output/src3814492293/src/github.com/apple/foundationdb/build_output/documentation/document_html_done
FAILED: documentation/document_html_done /codebuild/output/src3814492293/src/github.com/apple/foundationdb/build_output/documentation/document_html_done 
```

Maybe because backup_agent was not double quoted?

Reply:
Ah, wasn't aware of this, updated the file by using double anti quotes.